source/packages/integration-tests/src/support/greengrass2_provisioning_hooks.ts

```diff
@@ -29,7 +29,7 @@ import { container } from '../di/inversify.config';
 import { AUTHORIZATION_TOKEN } from '../step_definitions/common/common.steps';
 import { getAdditionalHeaders } from '../step_definitions/notifications/notifications.utils';
 
-import { logger } from '@awssolutions/simple-cdf-logger';
+// import { logger } from '@awssolutions/simple-cdf-logger';
```

Comment:
can we remove this as well?

Reply:
I kept these because it's annoying to re-add when you're debugging, but I didn't want them there by default. They're in the tests, so I don't think they hurt anything. Can remove if you feel strongly about it.

Reply:
I turned these back on and set them to the silly log level so they don't show in our current default log level.